docs/reference/query-languages/esql/kibana/definition/functions/date_parse.json

```diff
@@ -51,7 +51,7 @@
         {
           "name" : "options",
           "type" : "function_named_parameters",
-          "mapParams" : "{name='time_zone', values=[standard], description='Coordinated Universal Time (UTC) offset or IANA time zone used to convert date values in the query string to UTC.'}, {name='locale', values=[standard], description='The locale to use when parsing the date, relevant when parsing month names or week days.'}",
+          "mapParams" : "{name='time_zone', values=[standard], description='Coordinated Universal Time (UTC) offset or IANA time zone used to convert date values in the query string to UTC.', type=[keyword]}, {name='locale', values=[standard], description='The locale to use when parsing the date, relevant when parsing month names or week days.', type=[keyword]}",
```

Comment:
Huh, I've just noted that this isn't JSON. This feels like a weird format for Kibana, so I would ensure they can read this change.
And depending on their answer here, maybe migrate this to JSON? That's for another issue though